packages/react-components/react-nav-preview/src/components/NavDrawer/renderNavDrawer.tsx

```diff
@@ -1,22 +1,11 @@
 /** @jsxRuntime automatic */
 /** @jsxImportSource @fluentui/react-jsx-runtime */
 
-import { assertSlots } from '@fluentui/react-utilities';
-import type { NavDrawerState } from './NavDrawer.types';
+import { DrawerState, renderDrawer_unstable } from '@fluentui/react-drawer';
 import { NavContextValues } from '../NavContext.types';
```

Comment:
Should this one be a type import?
```suggestion
import type { NavContextValues } from '../NavContext.types';
```